src/button/config.js

```diff
@@ -157,7 +157,11 @@ export const BUTTON_CONFIG : ButtonConfig = {
 
     [ BUTTON_LABEL.CREDIT ]: {
         label:     `{ logo: ${ BUTTON_LOGO.PP } } { logo: ${ BUTTON_LOGO.PAYPAL } } { logo: ${ BUTTON_LOGO.CREDIT } }`,
-        logoLabel: `{ logo: ${ BUTTON_LOGO.PP } } { logo: ${ BUTTON_LOGO.PAYPAL } } { logo: ${ BUTTON_LOGO.CREDIT } }`,
+        logoLabel: ({ locale }) => {
+            return locale.country === COUNTRY.DE
```

Comment:
let's create an object and make this configurable instead of a condition?
Ex:
`const creditLogoForLocale = { 'DE': 'CREDIT_LOGO_DE }` and then read it off that?